packages/cli/src/utils/directory.test.ts

```diff
@@ -8,77 +8,210 @@ const pathsToMatch = (expected: string, desired: string) => {
   return expectedResolved === desiredResolved
 }
 
-describe('directory with base path', () => {
+describe('withBasePath as the current dir `.`', () => {
+  const basePath = '.'
+
   describe('coreDir', () => {
-    it('is the fastsstoreDir + core', () => {
-      expect(pathsToMatch(coreDir, './node_modules/@faststore/core')).toBe(true)
+    it('is the faststoreDir + core', () => {
+      const { coreDir: basedCoreDir } = withBasePath(basePath)
+
+      expect(pathsToMatch(basedCoreDir, './node_modules/@faststore/core')).toBe(true)
```

Comment:
suggestion (nit): just to match the others
```suggestion
      const { coreDir: coreDirWithBase } = withBasePath(basePath)

      expect(pathsToMatch(coreDirWithBase, './node_modules/@faststore/core')).toBe(true)
```